api/index.js

```diff
@@ -17,86 +18,71 @@ var utils = require('../utils');
 var app = express();
 
 const iberaServicesEndpoint = config.IBERA_SERVICES_ENDPOINT;
-const azureStorageConnectionString = config.AZURE_OI_STORAGE_CONNECTION_STRING;
+const azureStorageConnectionString = config.STORAGE_CONNECTION_STRING;
 
+const USER_TOKEN_HEADER_KEY = 'user-token';
 const CONTAINER_NAME = 'attachments';
 
+var azureBlobService = azureStorage.createBlobService(azureStorageConnectionString);
+
+//Will be implemented next,  https://amitu-ted.visualstudio.com/DefaultCollection/iBera/_workitems/edit/11
 async function getUserId(userToken) {
   return 'demo-user-001';
-  /* try {
-     var uri = documentServicesEndpoint + `/api/user`;
-     var result = await request({
-       method: 'GET',
-       uri,
-       headers: { 'User-Token': userToken },
-       json: true
-     });
-     console.log(`got response: ${util.inspect(result)}`);
-     return result;
-   }
-   catch (err) {
-     var errorMessage = `Could not retrieve userId from user token: ${userToken}`;
-     console.log(errorMessage);
-     throw new Error(errorMessage);
-   }
-   */
 }
 
 
-app.get('/config', async(req, res) => {
-
+app.get('/config', async (req, res) => {
   try {
-    var result = {
-      documentServiceUrl: config.DOCUMENT_SERVICES_ENDPOINT
-    }
+    var result = {documentServiceUrl: config.DOCUMENT_SERVICES_ENDPOINT}
 
     console.log(`sending configuration: ${util.inspect(result)}`);
     res.json(result);
   } catch (err) {
-    return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
-      error: err.message
-    });
+    return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({error: err.message});
   }
 });
 
 app.post('/attachment', async(req, res) => {
-  if (!req.headers['user-token']) {
-    return res.status(HttpStatus.BAD_REQUEST).json({
-      error: `user-token request header is missing`
-    });
+  var userToken = req.headers[USER_TOKEN_HEADER_KEY];
```

Comment:
All of the APIs in this file need to get the token and then resolve the user id based on the token.
Instead of adding this functionality in each API, we can create a middleware that will do that for all of the requests before even getting to the actual API handler.
Leave that as it is for now, and talk to me next week if you need help understanding what I meant.
And you can implement this as part of the authentication task.